vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/VGSCheckoutAddCardConfig.kt

```diff
@@ -115,5 +127,22 @@ class VGSCheckoutAddCardConfig private constructor(
         override fun newArray(size: Int): Array<VGSCheckoutAddCardConfig?> {
             return arrayOfNulls(size)
         }
+
```

Comment:
Please, add TODO label not to forget to remove it.

Reply:
Done